CSharp/Library/Microsoft.Bot.Builder.Azure/app.config

```diff
@@ -16,4 +16,34 @@
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
+  <system.serviceModel>
```

Comment:
these changes can be reverted now that the dependency moved to bot.builder.azure.logging